api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,770 @@
+    "            most_recent_value = str(most_recent_value)\n",
+    "        # Create string\n",
+    "        item_metric = item + \" \" + metric_name\n",
+    "        significance_headline = region+ \" \" + item_metric + \": \" + most_recent_value + \" \" + unit + \", \" + percent_change\n",
+    "    return significance_headline\n",
+    "\n",
+    "\n",
+    "def work_to_distribute(data):\n",
+    "    '''\n",
+    "    Method run by workers, this will run each of the tests in return_alerts\n",
+    "    '''\n",
+    "    string_add = \"\"\n",
+    "    data_line_2 = return_alerts(data)\n",
+    "    if(data_line_2!=\"\"):\n",
+    "        data_line_1 = string_formatting(data,-1)\n",
+    "        string_add += (data_line_1 + data_line_2 + \"\\n\")\n",
+    "    return string_add\n",
+    "\n",
+    "\n",
+    "def compute_amomalies(array, data_name):\n",
+    "    '''\n",
+    "    Takes array of data series and a category name and returns any alerts string formatted under a header\n",
+    "    '''\n",
+    "    string = \"\"\n",
+    "    string_add = \"\"\n",
+    "    string_list = []\n",
+    "    if(array != None):\n",
+    "        try:\n",
+    "            for ele in array:\n",
+    "                string_list.append(work_to_distribute(ele))\n",
+    "        except IndexError:\n",
+    "            print(\"IndexError in compute_amomalies for \" + data_name)\n",
+    "            return string_add\n",
+    "            \n",
+    "        for x in string_list:\n",
+    "            string_add += x\n",
+    "        if(string_add!=\"\"):\n",
+    "            string = data_name\n",
+    "        else:\n",
+    "            string = \"\"\n",
+    "\n",
+    "    data_statement = string + \" \\n\" + string_add\n",
+    "\n",
+    "    return data_statement\n",
+    "\n",
+    "\n",
+    "argentina_string = compute_amomalies(land_temperature_array, data_name)\n",
+    "print(\"argentina_string:\")\n",
+    "print(argentina_string)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NDVI Anomalies in Brazil\n",
+    "\n",
+    "Now that we have our alerts from Argentina about Land Temperature (If the cell above prints nothing, that means there were no alerts for significant temperature in Argentina), we will try brazil NDVI (Normalized Difference Vegetation Index) anomalies as well. Our data source for NDVI allows us to estimate the amount of chlorophyll in plants in a given region."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[102446, 102454, 102459, 102463, 102453, 102449, 102458, 102451, 102448, 102457, 102447, 102465, 102467, 102462, 102464, 102455, 102468, 102461, 102450, 102452, 102460, 102456, 102466]\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "# Tocantins Province in Brazil\n",
+    "tocantins_province_id = 10428\n",
```

Comment:
`tocantins_province_id` variable is not used. Looks like you meant to use it instead of `region_id` in the next line?

Reply:
yeah changed!